examples/ota-requestor-app/linux/README.md

```diff
@@ -21,6 +21,7 @@ following command line options are available for the OTA Requestor application.
 | -------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | -a, --autoApplyImage                                     | If supplied, apply the image immediately after download. Otherwise, the OTA update is complete after image download.                                                                                                                                                                                                                                                                                                                                                                                              |
 | -c, --requestorCanConsent \<true \| false\>              | Value for the RequestorCanConsent field in the QueryImage command. If not supplied, the value is determined by the driver.                                                                                                                                                                                                                                                                                                                                                                                        |
+| -d, --disableNotifyUpdateApplied                         | If supplied, disable sending of the NotifyUpdateApplied command. Otherwise, after successfully loading into the updated image, send the NotifyUpdateApplied command.                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
```suggestion
| -d, --disableNotifyUpdateApplied                         | If supplied, disable sending of the NotifyUpdateApplied command. Otherwise, after successfully loading into the updated image, send the NotifyUpdateApplied command. This should only be used for testing purposes.                                                                                                                                                                                                                                                                                                                                                                                             |
```

Reply:
All of these options are pretty much for testing purpose. I can leave a comment on top to specify so.